tests/test_waveform.py

```diff
@@ -20,8 +20,8 @@ def test_initial_buffer_length(self):
     def test_random_buffer_length(self, l1, l2):
         assume(l1 > 0 and l2 > 0)
         w = Waveform(np.ones(l1), np.ones(l2))
-        assert w.buffer_length % 16 == 0
-        assert w.buffer_length >= 32
+        assert w.buffer_length % 8 == 0
```

Comment:
The rules of minimal waveform length and granularity are different between the UHF and the HDAWG. can you distinguish between the two instruments?